### PR TITLE
Made the home page

### PR DESCRIPTION
**Pull request by BadPotato1007**
Dont think I messed up anything, but just check.. have to change the buttons function later to direct OAuth later...

### CODE REVIEW

**Review by carrotfarmer — CHANGES_REQUESTED** [review]
Put the images in a separate img folder not in the root dir

File: index.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="https://cdn.discordapp.com/attachments/1038360446383693904/1038361636211273728/pi_space_png.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>Pispace | Your one stop to success</title>
```
Comment (carrotfarmer) [inline index.html]: *You're

**Comment by carrotfarmer** [comment]
>  have to change the buttons function later to direct OAuth later...


and what do you mean by this?

**Comment by carrotfarmer** [comment]
image
aand please dont do this

**Comment by carrotfarmer** [comment]
Consider using a formatter like Prettier and the `pt` prop instead of using the `<br />` tag for Chakra components

**Comment by BadPotato1007** [comment]
> 

Its not using those images... we can delete them. its taking the imgs from urls

**Pull request closed without merging**